pkg/kubectl/resource/builder.go

```diff
@@ -227,12 +240,17 @@ func (b *Builder) LocalParam(local bool) *Builder {
 
 // Local will avoid asking the server for results.
```

Comment:
doc this means the returned Info will not have a RESTMapping or Client set

Reply:
> doc this means the returned Info will not have a RESTMapping or Client set

That's already doc'ed on the mapping and client fields

Reply:
not seeing that doc, but fine to include in follow up cleanup pulls. also, the doc on the Object field is drifting, should update once this is in it's final shape.